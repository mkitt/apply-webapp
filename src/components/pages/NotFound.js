// @flow
import React from 'react'
import glamorous from 'glamorous'
import AppContainment from '../elements/AppContainment'

export default () => (
  <AppContainment>
    <View>Not Found</View>
  </AppContainment>
)

const View = glamorous.div({
  fontWeight: 600,
})
