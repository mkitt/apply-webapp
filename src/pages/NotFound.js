// @flow
import React from 'react'
import glamorous from 'glamorous'
import AppContainment from '../components/elements/AppContainment'

export default () => (
  <AppContainment>
    <View>Not Found</View>
  </AppContainment>
)

const View = glamorous.div({
  fontWeight: 600,
})
