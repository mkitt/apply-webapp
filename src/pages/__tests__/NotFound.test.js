import React from 'react'
import renderer from 'react-test-renderer'
import NotFound from '../NotFound'

describe('/404', () => {
  let component

  beforeEach(() => {
    component = (
      <NotFound />
    )
  })

  it('shows the correct snapshot tree for the /404 page', () => {
    const tree = renderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
