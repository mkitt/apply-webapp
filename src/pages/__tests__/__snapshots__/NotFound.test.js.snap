// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/404 shows the correct snapshot tree for the /404 page 1`] = `
.glamor-1,
[data-glamor-1] {
  position: relative;
  max-width: 1392px;
  margin-right: auto;
  margin-left: auto;
  padding-right: 16px;
  padding-bottom: 16px;
  padding-left: 16px;
}

.glamor-0,
[data-glamor-0] {
  font-weight: 600;
}

<div
  className="glamor-1"
>
  <div
    className="glamor-0"
  >
    Not Found
  </div>
</div>
`;
